docs/exodus-intro.md

```diff
@@ -23,16 +23,16 @@ It uses:
 * Parallelism - can run compilations and tests in parallel.
 * Hermeticity - builds can access only what is declared and pre-fetched, so they run 'hermetically' without unnecesary noise from additional files. 
 
-### Multi-language environment
+### Multi-language environment 
 Here at Wix, we support different software languages and it's a huge advantage to be able to build multiple languages simultaneously. 
 
-### Scalability
+### Scalability 
 Bazel can handle multiple repositories or any size mono-repo. We are able to scale our codebase and our continuous integration system.
```

Comment:
due to build workers farm (read about bazel remote execution) we can scale out build actions to hunderds of workers simultaneously. so we have room to grow our code. >10MLOC in jvm languages